common/state/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "state"
+version = "0.1.0"
+authors = ["Chris Nixon <engineering@logdna.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+derivative = "2.2"
+futures = "0.3"
+rocksdb = "0.15"
+async-channel = "1.6"
```

Comment:
Just curious, why you chose async-channel instead of tokio's `broadcast`?

Reply:
By using async-channel it means that the module is runtime agnostic, at least unless we're running the tests.

It's a pretty minor nice-to-have given the rest of the agent all depends on Tokio, but in this case it was easy the module agnostic in case we want to use it somewhere else.

Reply:
ok, gotcha